### PR TITLE
feat(Roblox Forum): add new URLs + adds buttons

### PR DESCRIPTION
**Pull request by liechtenberg**
## Description 

adds new URLs to it (mostly for people who have permission to moderate the forum or subforums specifically) & also adds buttons

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

image

</details>

### CODE REVIEW

**Comment by sybetjeh** [comment]
why arent the checks like checking it?

**Comment by sybetjeh** [comment]
Also, how do you fix this? This only happens when there's a button, is it not possible to do Starttimestamp w/ buttons? https://cdn.bananahosting.xyz/6ba2b9e3

File: websites/R/Roblox Forum/presence.ts [hunk 1]
```diff
@@ -25,8 +30,20 @@ presence.on("UpdateData", async () => {
 		presenceData.state = `Profile: ${document
 			.querySelector("meta[property='og:title']")
 			.getAttribute("content")}`;
-	} else if (document.location.pathname.startsWith("/whats-new/"))
-		presenceData.details = "Browsing latest content";
+		presenceData.buttons = [
+			{
+				label: "View profile",
```
Comment (EncryptedDev) [inline websites/R/Roblox Forum/presence.ts:35]:
```suggestion
				label: "View Profile",
```

File: websites/R/Roblox Forum/presence.ts [hunk 2]
```diff
@@ -15,6 +14,12 @@ presence.on("UpdateData", async () => {
 		presenceData.state = `Thread: ${document
 			.querySelector("meta[property='og:title']")
 			.getAttribute("content")}`;
+		presenceData.buttons = [
+			{
+				label: "View thread",
```
Comment (EncryptedDev) [inline websites/R/Roblox Forum/presence.ts:19]:
```suggestion
				label: "View Thread",
```

File: websites/R/Roblox Forum/presence.ts [hunk 3]
```diff
@@ -25,8 +30,20 @@ presence.on("UpdateData", async () => {
...
 			.getAttribute("content")}`;
-	} else if (document.location.pathname.startsWith("/whats-new/"))
-		presenceData.details = "Browsing latest content";
+		presenceData.buttons = [
+			{
+				label: "View profile",
+				url: document.location.href,
+			},
+		];
+	} else if (document.location.pathname.startsWith("/admin.php"))
```
Comment (EncryptedDev) [inline websites/R/Roblox Forum/presence.ts:39]: Combine this else if with the others by using an OR operator

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/R/Roblox Forum/dist/metadata.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
 		"en": "Welcome to Roblox Forum. Join the discussion here for the best help available. Scripting, Building, Game Creating, or just for fun. We have it all."
 	},
 	"url": "www.robloxforum.com",
-	"version": "1.0.0",
+	"version": "1.0.1",
```
Comment (EncryptedDev) [inline websites/R/Roblox Forum/dist/metadata.json:12]:
```suggestion
	"version": "1..1.0",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/R/Roblox Forum/dist/metadata.json (hunk 4 above)
Comment (sybetjeh) [reply websites/R/Roblox Forum/dist/metadata.json]: done

File: websites/R/Roblox Forum/presence.ts (hunk 2 above)
Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: done

File: websites/R/Roblox Forum/presence.ts (hunk 1 above)
Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: done

File: websites/R/Roblox Forum/presence.ts (hunk 3 above)
Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: i tried using || but i couldn't get that to work, I tried using && but only works with the first thing that starts

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: nvm on that, i removed startsWith and it worked. i have no idea if this is the most efficient way of doing this, so lmk what u think

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: yeah this deff isnt the way how I should do it, it errors on the checks -- but I have no idea what/or how I could fix this?

image

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: image
I have this locally, but only the first one works. The other 2 don't, don't know why

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: tbf the amin.php one isn't needed, so i removed it

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: image
if i use 1 = it works both, but it causes to refresh the page infinitely. if u use 2 == only 1 works

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: i'm stuck on this one, id you could give me a push in the right direction i'd appreciate it a lot

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts]: (also using startsWith doesn't work at all)

**Comment by EncryptedDev** [comment]
Run `yarn format`

**Comment by sybetjeh** [comment]
> Run `yarn format`

okey

**Comment by sybetjeh** [comment]
@EncryptedDev i did it, but it doesn't work tho. look at the unresolved convo, I tried doing it

File: websites/R/Roblox Forum/presence.ts [hunk 5]
```diff
@@ -25,8 +30,16 @@ presence.on("UpdateData", async () => {
...
 			.getAttribute("content")}`;
-	} else if (document.location.pathname.startsWith("/whats-new/"))
-		presenceData.details = "Browsing latest content";
+		presenceData.buttons = [
+			{
+				label: "View Profile",
+				url: document.location.href,
+			},
+		];
+	} else if (window.location.pathname === ("/reports/" && "/approval-queue/"))
```
Comment (EncryptedDev) [inline websites/R/Roblox Forum/presence.ts:39]: This seems wrong or is it correct?

Comment (sybetjeh) [reply websites/R/Roblox Forum/presence.ts:39]: do you read the things i said? i tried everything, it doesn't work. i might as well remove it

File: websites/R/Roblox Forum/presence.ts (hunk 3 above)
Comment (EncryptedDev) [reply websites/R/Roblox Forum/presence.ts]: I’m on holiday so this is hard to type out syntactically 

if(pathname.includes(“admin”) || pathname.includes(“approval-queue”) presenceData.details = “Moderation shenanigans”

**Comment by EncryptedDev** [comment]
Resolve CI

**Comment by sybetjeh** [comment]
@EncryptedDev done, resolved

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**